src/build_lds.ts

```diff
@@ -1,22 +1,20 @@
+/**
+ * @file Build script for the language server.
+ * @author Marten Lohstroh <marten@berkeley.edu>
+ */
+
 'use strict';
 
 import * as fs from 'fs'
 import * as path from 'path'
 import * as rimraf from 'rimraf'
 import simpleGit, { SimpleGit } from 'simple-git'
 import { Command, OptionValues } from 'commander'
-import { Config } from './config'
+import * as config from './config'
```

Comment:
Why is this better?

Reply:
You might have a similar reaction to the reactor-ts code base. I would enjoy receiving some code review from you on that :-)